OrbitCore/TracepointEventBuffer.cpp

```diff
@@ -51,4 +47,18 @@ TracepointEventBuffer::GetTracepointsOfThread(int32_t thread_id) const {
   return it->second;
 }
 
-Mutex& TracepointEventBuffer::mutex() { return mutex_; }
\ No newline at end of file
+void TracepointEventBuffer::ForEachTracepointEventPerThread(
```

Comment:
This is called `ForEachTracepoint...`, but then the `action` takes the whole map and the responsibility of iterating over the tracepoints is left to `action` itself. This seems wrong. Am I missing something? I would expect `action` to be simply `std::function<void(const orbit_client_protos::TracepointEventInfo&)>` and this function to iterate on threads *and* and events.

Reply:
Thank you!